sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -776,8 +776,8 @@ export class ServiceBusMessageImpl implements ServiceBusReceivedMessage {
     if (msg.body) {
       this.body = defaultDataTransformer.decode(msg.body);
     }
-    // TODO: _amqpAnnotatedMessage is already being populated in fromRheaMessage(), no need to do it twice
-    this._amqpAnnotatedMessage = AmqpAnnotatedMessage.fromRheaMessage(msg);
+    // TODO: _rawAmqpMessage is already being populated in fromRheaMessage(), no need to do it twice
```

Comment:
For this TODO, it is a typescipt issue that the assignments in `Object.assign()` are not properly recognized and complains about not assigning `_rawAmqpMessage`.

Related: https://github.com/microsoft/TypeScript/issues/26792